server/tests/api_v5/anonymous/test_pki_enrollment_submit.py

```diff
@@ -292,6 +321,8 @@ async def test_anonymous_pki_enrollment_submit_email_already_used(
             human_handle=coolorg.alice.human_handle,
         ).dump(),
     )
+
+    # 3. ...
```

Comment:
something is missing ?

Reply:
Not for me, the text would have been `check for email already submitted` but I found that redundant with the assert below